src/Identity/Controllers/AccountsController.cs

```diff
@@ -67,6 +84,24 @@ public async Task<RegisterResponseModel> PostRegister([FromBody] RegisterRequest
         throw new BadRequestException(ModelState);
     }
 
+    [RequireFeature(FeatureFlagKeys.EmailVerification)]
+    [HttpPost("register/send-verification-email")]
+    public async Task<IActionResult> PostRegisterSendVerificationEmail([FromBody] RegisterSendVerificationEmailRequestModel model)
+    {
+        var refEvent = new ReferenceEvent(ReferenceEventType.SignupEmailSubmit, null, _currentContext);
+        await _referenceEventService.RaiseEventAsync(refEvent);
```

Comment:
We may want to fire this after sending the email, just in case

Reply:
Moved!